src/components/timers/Countdown.tsx

```diff
@@ -1,3 +1,62 @@
-const Countdown = () => null;
+import React, { useState, useEffect } from 'react';
 
-export default Countdown;
+const Countdown: React.FC = () => {
+  // track remaining time in sec
+  const [time, setTime] = useState(150); // deliverable eg: starting from 2 min 30 sec
+  const [isRunning, setIsRunning] = useState(false);
+
+ // useEffect handle timer interval when timer is running
+  useEffect(() => {
+    if (isRunning && time > 0) {
+      // decrement time by 1 sec
+      const interval = setInterval(() => {
+        setTime(prevTime => prevTime - 1);
+      }, 1000);
+      return () => clearInterval(interval);
+    } else if (time === 0) {
+      setIsRunning(false);
+    }
+  }, [isRunning, time]);
+
+  // Below Functions: start/resume, pause, reset, fast forward timer
+  const handleStart = () => {
+    setIsRunning(true);
+  };
+
+  const handlePause = () => {
+    setIsRunning(false);
+  };
+
+  const handleReset = () => {
+    setIsRunning(false);
+    setTime(150);
```

Comment:
these hardcoded numbers should be at least moved out to a constant so if you need to change them at least have it in a single place. a better approach would be to pass them in as props.